File: Algothrim/project_euler/2/2.cpp
#include <iostream>

int main(void)
{
    unsigned long a = 1;
    unsigned long b = 2;
    unsigned long sum = a + b;
    unsigned long tmp_b;

    while (b < 4000000) {
        tmp_b = b;
        b = a + tmp_b;
        a = tmp_b;

        sum += b;
    }

    std::cout << "sum is: " << sum << std::endl;

    return 0;
}

File: Algothrim/project_euler/2/2.py
#!/usr/bin/env python3
# encoding: utf-8

a = 1
b = 2
sum_num = a+b

while b < 4000000:
    tmp_b = b
    b = a + tmp_b
    a = tmp_b
    sum_num += b

print("sum is:", sum_num)

